lib/render.js

```diff
@@ -79,7 +83,11 @@ var render = module.exports = function (html, options) {
   if (options.highlightSyntax) parser.use(codeWrap)
   if (options.serveImagesWithCDN) parser.use(cdnImages, {package: options.package})
 
-  return githubLinkify(parser).render(html)
+  return githubLinkify(parser)
+}
+
+render.renderPackageDescription = function (description) {
```

Comment:
oh yeah, and this has absolutely nothing to do with the packagize stuff!

and finally, the only reason it shows up in the diff here is because I moved it up earlier in the file so it grouped the module exports together 😕 

yay legacy